release/Magarena/scripts/Felhide_Spiritbinder.groovy

```diff
@@ -0,0 +1,47 @@
+def choice = new MagicTargetChoice("another target creature");
+
+def TypeStatic = new MagicStatic(MagicLayer.Type) {
+    @Override
+    public int getTypeFlags(final MagicPermanent permanent,final int flags) {
+        return flags | MagicType.Enchantment.getMask();
+    }
+};
+
+def Enchantment = {
+    final MagicPermanent perm ->
+    final MagicGame game = perm.getGame();
+    game.doAction(new AddStaticAction(perm, TypeStatic));
+}
+
+[
+    new ThisBecomesUntappedTrigger() {
+        @Override
+        public MagicEvent executeTrigger(final MagicGame game, final MagicPermanent permanent, final MagicPermanent untapped) {
+            return new MagicEvent(
+                permanent,
+                new MagicMayChoice(
+                    new MagicPayManaCostChoice(MagicManaCost.create("{1}{R}")),
+                choice,
+                ),
+                this,
+                "PN may\$ pay {1}{R}\$. If PN does, PN creates a token that's a copy of another target creature\$, " +
+                "except it's an enchantment in addition to its other types. It gains haste. Exile it at the beginning of the next end step."
+            );
+        }
+        @Override
+        public void executeEvent(final MagicGame game, final MagicEvent event) {
+            if (event.isYes()) {
+                event.processTargetPermanent(game, {
+                    game.doAction(new PlayTokenAction(
+                        event.getPlayer(),
+                        it,
+                        Enchantment,
+                        MagicPlayMod.HASTE_UEOT,
```

Comment:
should be MagicPlayMod.HASTE (oracle text says "It gains haste", not "It gains haste until end of turn")

Reply:
Fixed. As the token is exiled at EOT, it was easy to miss the difference (there are some rare cases though where it would make a difference, i.e. if you create a copy of the token)